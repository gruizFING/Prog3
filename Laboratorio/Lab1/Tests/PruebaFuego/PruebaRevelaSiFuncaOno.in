VAR =
VAR                                           =
VAR                                                        =1
VAR                                                              =         13
EXPR =
EXPR                                         =
EXPR                                                   =        1
EXPR                                             =       variable1+variable2
EXPR                       expr1        =           variable1/variable2
EVAL expr1
EVAL expr1/90
EVAL expr1+variable1
VAR                         variable1               =               60
VAR                                  variable2               =             30
EVAL                                expr1
EVAL                                                 expr1/90
EVAL                                                        expr1+variable1
EVAL         variable2-variable1
VAR          variable3
VAR          variable3 = 70
SAVE                                                guardarsub0
VAR          variable3             =           75
VAR          variable3                               =80
VAR          variable4             =             70
VAR          variable5                  =          60
VAR               variable6                =      80
VAR         variable7           =      56
VAR                 variable7                     =        40
VAR           variable8            =      0
VAR                            variable9          =     700
VAR              variable10    = 68
VAR =
VAR          =
VAR              =
VAR      variable11       = 400
VAR variable10 = 1000
VAR variable9 = 2000
EVAL             variable10
EVAL             variable9
EVAL             variable1+variable2+variable3
EVAL             variable1+variable2+variable3/variable8
EXPR
EXPR =
EXPR     =   variable1+variable2
EXPR     expr2            =             900
EXPR               expr3       =            90000
EXPR expr4=((((((((variable1*variable1*variable1*variable2*variable2*variable3)/variable6)/variable1)/variable1)/variable1)/variable1)/variable1)/variable1)
EVAL expr4
EVAL expr3
EVAL expr2
EVAL expr5
SAVE        guardar0
EXPR              expr5        =     variable10
EXPR                     expr6       = (variable10+variable9)/variable8
EXPR expr7 = ((((variable1*variable2)*variable3)*variable4)/variable5)/variable10
EXPR expr8 = ((((variable1/variable2)/variable3)/variable4)/variable5)/variable6
EXPR expr9 = variable1
EXPR expr10 = variable8
EVAL expr7
EVAL expr8
EVAL expr9
EVAL expr10
EVAL expr7-expr8
EVAL expr8-expr10
EVAL ((((expr3-expr1)/expr4)*expr7)/10)*variable1
EVAL ((((expr8-expr10)/expr9)*expr3)/90)*variable1
EVAL expr10*expr6
SAVE                        guardar1
LOAD                 guardar1
SAVE                       guardar2
LOAD                 guardar2
SAVE                       guardar3
LOAD                 guardar3
SAVE                       guardar4
SAVE                       guardar4
SAVE                       guardar4
SAVE                       guardar4
SAVE                       guardar4
SAVE                       guardar4
SAVE                       guardar4
SAVE                       guardar4
SAVE                       guardar4
LOAD                       guardar4
LOAD                       guardar4
LOAD                       guardar4
LOAD                       guardar4
LOAD                       guardar4
LOAD                       guardar4
LOAD                       guardar4
EVAL                 expr7
EVAL                       expr8
EVAL                 expr9
EVAL expr10
EVAL expr7-expr8
EVAL expr8-expr10
EVAL ((((expr3-expr1)/expr4)*expr7)/10)*variable1
EVAL ((((expr8-expr10)/expr9)*expr3)/90)*variable1
EVAL expr10*expr6
EVAL                    expr10*expr6
EVAL ((((((expr4*expr4)/expr4)/expr4)*expr4)*expr5)*expr1)
LOAD                                  guardarsub0
EVAL                                expr1
EVAL                                                 expr1/90
EVAL                                                        expr1+variable1
EVAL         variable2-variable1
EVAL variable3
SAVE guardar5
LOAD guardar0
EVAL expr4
EVAL expr3
EVAL expr2
EVAL expr5
LOAD guardar4
EVAL expr5
EVAL expr4
EVAL ((((expr3-expr1)/expr4)*expr7)/10)*variable1
EVAL ((((expr8-expr10)/expr9)*expr3)/90)*variable1
LOAD guardarsub0
LOAD guardar0
LOAD guardar1
LOAD                 guardar2
LOAD                 guardar3
LOAD                 guardar4
LOAD guardarsub0
LOAD guardar0
LOAD guardar1
LOAD                 guardar2
LOAD                 guardar3
LOAD                 guardar4
LOAD guardarsub0
LOAD guardar0
LOAD guardar1
LOAD                 guardar2
LOAD                 guardar3
LOAD                 guardar4
LOAD guardarsub0
LOAD guardar0
LOAD guardar1
LOAD                 guardar2
LOAD                 guardar3
LOAD                 guardar4
LOAD guardarsub0
LOAD guardar0
LOAD guardar1
LOAD                 guardar2
LOAD                 guardar3
LOAD                 guardar4
LOAD guardarsub0
LOAD guardar0
LOAD guardar1
LOAD                 guardar2
LOAD                 guardar3
LOAD                 guardar4
LOAD guardarsub0
LOAD guardar0
LOAD guardar1
LOAD                 guardar2
LOAD                 guardar3
LOAD                 guardar4
LOAD guardarsub0
LOAD guardar0
LOAD guardar1
LOAD                 guardar2
LOAD                 guardar3
LOAD                 guardar4
EVAL variable1
EVAL variable2
EVAL variable3
EVAL variable4
EVAL variable5
EVAL variable6
EVAL variable7
EVAL variable8
EVAL variable9
EVAL variable10
EVAL expr1
EVAL expr2
EVAL expr3
EVAL expr4
EVAL expr5
EVAL expr6
EVAL expr7
EVAL expr8
EVAL expr9
EVAL expr10
LOAD guardarsub0
VAR sitefuncoesporqueandasbien = 10
EVAL sitefuncoesporqueandasbien
SALIR