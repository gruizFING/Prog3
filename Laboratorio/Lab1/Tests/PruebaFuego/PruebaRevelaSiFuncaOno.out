Bienvenido a el mini-sistema:
> ERROR: Nombre de variable vacio.
> ERROR: Nombre de variable vacio.
> ERROR: Nombre de variable vacio.
> ERROR: Nombre de variable vacio.
> ERROR: La expresion no es valida.
> ERROR: La expresion no es valida.
> ERROR: Nombre de expresion vacio.
> ERROR: Nombre de expresion vacio.
> OK. expr1 agregada.
> ERROR: Falta definir variable o expresion.
> ERROR: Falta definir variable o expresion.
> ERROR: Falta definir variable o expresion.
> OK. variable1 agregada.
> OK. variable2 agregada.
> OK. 2.00
> OK. 0.02
> OK. 62.00
> OK. -30.00
> OK. variable3 agregada.
> OK. variable3 agregada.
> OK. Datos guardados.
> OK. variable3 agregada.
> OK. variable3 agregada.
> OK. variable4 agregada.
> OK. variable5 agregada.
> OK. variable6 agregada.
> OK. variable7 agregada.
> OK. variable7 agregada.
> OK. variable8 agregada.
> OK. variable9 agregada.
> OK. variable10 agregada.
> ERROR: Nombre de variable vacio.
> ERROR: Nombre de variable vacio.
> ERROR: Nombre de variable vacio.
> ERROR: Se alcanzo la maxima cantidad de variables.
> OK. variable10 agregada.
> OK. variable9 agregada.
> OK. 1000.00
> OK. 2000.00
> OK. 170.00
> ERROR: Division por cero.
> ERROR: La expresion no es valida.
> ERROR: La expresion no es valida.
> ERROR: Nombre de expresion vacio.
> OK. expr2 agregada.
> OK. expr3 agregada.
> OK. expr4 agregada.
> OK. 0.00
> OK. 90000.00
> OK. 900.00
> ERROR: Falta definir variable o expresion.
> OK. Datos guardados.
> OK. expr5 agregada.
> OK. expr6 agregada.
> OK. expr7 agregada.
> OK. expr8 agregada.
> OK. expr9 agregada.
> OK. expr10 agregada.
> OK. 168.00
> OK. 0.00
> OK. 60.00
> OK. 0.00
> OK. 168.00
> OK. 0.00
> OK. 21772316160.00
> OK. 0.00
> ERROR: Division por cero.
> OK. Datos guardados.
> OK. Datos cargados.
> OK. Datos guardados.
> OK. Datos cargados.
> OK. Datos guardados.
> OK. Datos cargados.
> OK. Datos guardados.
> OK. Datos guardados.
> OK. Datos guardados.
> OK. Datos guardados.
> OK. Datos guardados.
> OK. Datos guardados.
> OK. Datos guardados.
> OK. Datos guardados.
> OK. Datos guardados.
> OK. Datos cargados.
> OK. Datos cargados.
> OK. Datos cargados.
> OK. Datos cargados.
> OK. Datos cargados.
> OK. Datos cargados.
> OK. Datos cargados.
> OK. 168.00
> OK. 0.00
> OK. 60.00
> OK. 0.00
> OK. 168.00
> OK. 0.00
> OK. 21772316160.00
> OK. 0.00
> ERROR: Division por cero.
> ERROR: Division por cero.
> OK. 8.33
> OK. Datos cargados.
> OK. 2.00
> OK. 0.02
> OK. 62.00
> OK. -30.00
> OK. 70.00
> OK. Datos guardados.
> OK. Datos cargados.
> OK. 0.00
> OK. 90000.00
> OK. 900.00
> ERROR: Falta definir variable o expresion.
> OK. Datos cargados.
> OK. 1000.00
> OK. 0.00
> OK. 21772316160.00
> OK. 0.00
> OK. Datos cargados.
> OK. Datos cargados.
> OK. Datos cargados.
> OK. Datos cargados.
> OK. Datos cargados.
> OK. Datos cargados.
> OK. Datos cargados.
> OK. Datos cargados.
> OK. Datos cargados.
> OK. Datos cargados.
> OK. Datos cargados.
> OK. Datos cargados.
> OK. Datos cargados.
> OK. Datos cargados.
> OK. Datos cargados.
> OK. Datos cargados.
> OK. Datos cargados.
> OK. Datos cargados.
> OK. Datos cargados.
> OK. Datos cargados.
> OK. Datos cargados.
> OK. Datos cargados.
> OK. Datos cargados.
> OK. Datos cargados.
> OK. Datos cargados.
> OK. Datos cargados.
> OK. Datos cargados.
> OK. Datos cargados.
> OK. Datos cargados.
> OK. Datos cargados.
> OK. Datos cargados.
> OK. Datos cargados.
> OK. Datos cargados.
> OK. Datos cargados.
> OK. Datos cargados.
> OK. Datos cargados.
> OK. Datos cargados.
> OK. Datos cargados.
> OK. Datos cargados.
> OK. Datos cargados.
> OK. Datos cargados.
> OK. Datos cargados.
> OK. Datos cargados.
> OK. Datos cargados.
> OK. Datos cargados.
> OK. Datos cargados.
> OK. Datos cargados.
> OK. Datos cargados.
> OK. 60.00
> OK. 30.00
> OK. 80.00
> OK. 70.00
> OK. 60.00
> OK. 80.00
> OK. 40.00
> OK. 0.00
> OK. 2000.00
> OK. 1000.00
> OK. 2.00
> OK. 900.00
> OK. 90000.00
> OK. 0.00
> OK. 1000.00
> ERROR: Division por cero.
> OK. 168.00
> OK. 0.00
> OK. 60.00
> OK. 0.00
> OK. Datos cargados.
> OK. sitefuncoesporqueandasbien agregada.
> OK. 10.00
> 